new/LIT7416QeneLazeMaya.xml

```diff
@@ -34,7 +34,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>The author of the qǝne is not indicated in the manuscript. The type of the qǝne is indicated as śǝllāse.</p>
+                <p>The author of the qǝne is not indicated in the manuscript. The type of the qǝne is indicated as 
+                    <foreign xml:lang="gez-tr">śǝllāse</foreign>.</p>
```

Comment:
If I remember correctly, the decision was to use gez for cases like this and gez-tr rather when we are citing what someone else transliterated.